### ISSUE TITLE
[MAS1.4.11] [Visual Requirements-Open Bot Dialog] Color contrast ratio of Edit field and button boundary's is less than 3:1.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1963]
**Actual Result:**
The color contrast ratio of "Bot URL, Microsoft App ID and Microsoft App password" edit box boundary is less than 3:1.
As well as the color contrast ratio of "Cancel" button is also less than 3:1.

**Expected Result:**
Color contrast ratio should be greater than or equal to 3:1 for border outline with respective background color

**Note:** 
This issue exits for throughout application for all edit fields and buttons control.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the Welcome tab and reach at "Open Bot" button and activate it.
4. "Open a Bot" dialog gets appeared, navigate to "Bot URL/Microsoft App ID/Microsoft App password" edit field.
5. Calculate the color contrast ratio and observe the issue.
6. Observe the same for "Cancel" button.

**User Impact:**
Low Vision users will be impacting if the Color contrast ratio of the boundary is less than 3:1.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=6d787213-d391-4513-b643-14ad134a9aec](url)

**Attachment:**
![CCA of button and edit fields are failed](https://user-images.githubusercontent.com/55279608/68103894-99621100-fefe-11e9-9e5a-5cd90e50e44c.png)

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is still not fixed. Please look into the screenshot below:

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-94878
Attachment:
![Regress_10-12-19](https://user-images.githubusercontent.com/55279608/70513685-19ac0f80-1b58-11ea-9b47-f20a981c951d.png)

**Comment by tonyanziano** [comment]
I have just tested this on the latest nightly, and I see the correct colors that pass the color contrast ratio:

![image](https://user-images.githubusercontent.com/3452012/70567252-6e488e00-1b4a-11ea-8e9a-dcf57ab8f8f7.png)

Are you sure you are on version v4.6.0-94878?

**Comment by ShikhaMishra11** [comment]
@tonyanziano Yes we checked on the latest build. As per color code i.e. #888 on #F4F4F4, the CCA is passes the 3:1 ratio. So, we are good to close the issue.
